plugins/commands/compare/src/main/kotlin/CompareCommand.kt

```diff
@@ -155,12 +155,27 @@ class CompareCommand : OrtCommand(
 
                 throw ProgramResult(1)
             }
+
+            CompareMethod.SEMANTIC_DIFF -> {
+                echo(
+                    "The '${CompareMethod.SEMANTIC_DIFF}' compare method is not fully implemented. Some properties" +
+                        " may not be taken into account in the comparison."
+                )
+
+                if (resultA == resultB) {
+                    echo("The ORT results are the same.")
+                    throw ProgramResult(0)
+                }
+
+                throw ProgramResult(1)
+            }
         }
     }
 }
 
 private enum class CompareMethod {
-    TEXT_DIFF
+    TEXT_DIFF,
+    SEMANTIC_DIFF
```

Comment:
Nit: Should we maintain alphabetical sorting here (and also in the `when` above)?